cmake/ConfigureCompiler.cmake

```diff
@@ -183,7 +193,12 @@ else()
       -Wno-undefined-var-template
       -Wno-unused-value
       -Wno-tautological-pointer-compare
+      -Wno-register
```

Comment:
Are we using the `register` keyword in any C++ code? If so, we should probably stop using it. If not and this is coming from C code, is it possible to compile the C files such that they don't produce this warning? I wouldn't have expected to get a warning about using this keyword in C code.

Reply:
A quick grep shows that at least zlib seems to use this. I agree, removing this warning doesn't seem to be a good idea. We already compile sqlite as its own static library - we might want to do this with all external dependencies.

Reply:
Register keyword is fine for CLANG (when compiling locally). It is only a problem and produces an error when ccache/distcc is enabled.